### ISSUE TITLE
[Feature] Relative performer age for scene

### ISSUE THREAD

**Issue by Anon247** [stashapp/stash#430]
It would be really useful to be able to have the information on the age of the performer at the time a scene was shot. 

Not only is it just interesting information to see on a specific scene, but this would also enable powerful search functionality. A user would be able to search across an entire library for performers at a certain relative age no matter whether the scene was shot 30 years ago or yesterday.

Hopefully this will be relatively easy to implement too as the scene date and the performer birthdays are already recorded.

**Comment by bnkai** [comment]
This is already partially done, when you go to a scene and look at the perfomers tab it shows relative age to scene.
What is missing is the relative age of the performer to the scene as an extra filter for the scenes page.

**Comment by Anon247** [comment]
You're right, I completely missed it, must've only looked at scenes without both the scene date and the performer birthday. Great feature to have, glad it's already there 👍

**Comment by VirtaShops** [comment]
This is what I was going to request but found this request. I love that it shows the performers age at the time the scene was made, but building this into search/filters would make it so much more useful. Right now I have to click on each individual scene, see the performers age, go to edit, then create a tag representing their age: 18, 21, etc. All the info is there for this to be done automatically, would be amazing if this ability could be added!

**Comment by gitgiggety** [comment]
As it happens to be I'm currently working on a way to filter content (scenes, galleries, images) based on performers properties. So for example hair color, the alias text, tags (obviously currently available already). Filtering on age would obviously be part of this as well, although it requires either a dirty hack or a lot more work to implement.

But the most difficult part would be adding it to the UI. As IMO the current UI doesn't really allow for a "Filter by performers which are 25 years old, have black hair, and blue eyes". In the current UI I would then have to add a "Performer properties" or whatever to the list of filters, and then one way or another allow adding or setting the performer filters for which you could set "equals", "not equals" etc once more. So that really wouldn't work.

**Comment by sleetx** [comment]
> Wouldn't it make more sense to store the birthdate of the performer and the filming date of the scene? Then any sort of relative age can be computed.

As the original poster mentioned, both of those data points are already stored, and the calculation is displayed on the scene page. It just needs to be added as a search filter.
